.changeset/three-steaks-exist.md

```diff
@@ -0,0 +1,5 @@
+---
+"@astrojs/react": patch
+---
+
+feat: add experimentalDisableStreaming to react integration (#1)
```

Comment:
```suggestion
Add support for disabling streaming in React integration
```

Reply:
@florian-lefebvre Where would I place a code snippet in this repo? How would I go about opening a docs PR? Any advice or direction would be most appreciated.

Reply:
Ah I figured it out! https://github.com/withastro/docs/pull/10761